src/templates/DatasetSearch/DatasetSearch.tsx

```diff
@@ -137,7 +137,7 @@ const DatasetSearch = (props: DatasetSearchPageProps) => {
       startingNumber: Number(totalItems) >= startingNumber ? startingNumber : 0,
       endingNumber: Number(totalItems) < endingNumber ? Number(totalItems) : endingNumber,
     });
-    if (totalItems && totalItems <= 0 && currentResultNumbers !== null) {
+    if (totalItems <= 0 && currentResultNumbers !== null) {
```

Comment:
when `totalItems` equals `0` it evaluates as falsely. Removing this evaluation allows `No Results Found` to render when the value is `0`.